content/ecosystem/build-vs-buy/frameworks/dyrectorio.md

```diff
@@ -0,0 +1,91 @@
++++
+title="dyrector.io"
+url="/frameworks/dyrectorio"
++++
+
+# dyrector.io
+
+**Claim:** Deployment and configuration management without specialists.
+
+**Focus:** dyrector.io allows developers to use Kubernetes and Docker in an abstract manner. By using dyrector.io, non-specialists can execute multi-instance deployments and manage app configurations & secrets regardless of the cloud provider your team uses to host your applications.
+
+**Website:**[ dyrector.io](https://dyrector.io/)
```

Comment:
`https://dyrector.io/` throws a SSL error, and `http://dyrector.io/` gives a "404 Web Site not found"

Reply:
Yea, thanks for pointing this out. I am working on it right now.

Reply:
https://dyrector.io/features  gives a 404 too, "This site can’t be reached"

Reply:
Because we had an old landing page, now redirects the landing to our docs. Could you please check the URLs right now?